test/saithriftv2/convert_header.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import argparse
+
+"""
+Convert the generated code in sai_header.py.
+For the generated code from c++ enum, convert it to python enum class.
+
+the code like
+    ```C++
+    /**
+     * @brief SAI common API type
+     */
+    typedef enum _sai_common_api_t
+    {
+        SAI_COMMON_API_CREATE      = 0,
+        SAI_COMMON_API_REMOVE      = 1,
+        ...
+    } sai_common_api_t;
+    ```
+    original converted by ctypesgen
+    ```
+    enum__sai_common_api_t = c_int# /usr/include/sai/saitypes.h: 183
+    
+    SAI_COMMON_API_CREATE = 0# /usr/include/sai/saitypes.h: 183
+    
+    SAI_COMMON_API_REMOVE = 1# /usr/include/sai/saitypes.h: 183
+    ...
+    ```
+    new code
+    ```
+    class sai_common_api(SAIEnum):
+    
+        SAI_COMMON_API_CREATE = 0# /usr/include/sai/saitypes.h: 183
+    
+        SAI_COMMON_API_REMOVE = 1# /usr/include/sai/saitypes.h: 183
+    ...
+    
+    SAI_COMMON_API_CREATE = sai_common_api.SAI_COMMON_API_CREATE
+    SAI_COMMON_API_REMOVE = sai_common_api.SAI_COMMON_API_REMOVE
+    ...
+    ```
+    
+"""
+
+def parse_param():
+    '''
+    Parse param.
+    '''
+    parser = argparse.ArgumentParser(
+        description="""
+        Convert python file after ctypesgen convert.
+        """
+    )
+
+    parser.add_argument(
+        "-i", type=str, dest="input_file",
+        help="input file name", required=True)
+    parser.add_argument(
+        "-o", type=str, dest="output_file",
+        help="output file name", required=True)
+    return parser.parse_args()
+
+def import_and_enum_define():
+    """
+    import and add the enum class definitation.
+    """
+    def_str = "import enum\r\nclass SAIEnum(enum.IntEnum):\r\n    def __str__(self):\r\n        return super().__str__().split(\".\")[1]\r\n"            
```

Comment:
```suggestion
    def_str = """
import enum
class SAIEnum(enum.IntEnum):
    def __str__(self):
        return super().__str__().split(".")[1]
"""
```
Let‘s use triple quotes to create a multiline string, it is more easy to read.

Reply:
great suggestion